dc.sh

```diff
@@ -29,6 +43,24 @@ function up {
   docker ps -a
 }
 
+function publish() {
+    if [ -z ${PATH_TO_CONTRACTS+x} ]; then
+        echo "Provide a valid path to the contracts directory you wish to publish using the -p argument."
+        exit
+    fi
+
+   source ./service/docker/bootstrap.env
+    export FULL_PATH=$(realpath $PATH_TO_CONTRACTS)
+
+    if [[ -d "$FULL_PATH" ]]; then
+        pushd $PATH_TO_CONTRACTS > /dev/null
+        ./gradlew p8eClean p8eBootstrap --info
```

Comment:
Consider adding the p8e check task, along with the local maven publish task - we’ll need the newly created contract hashes to be able to execute these locally